packages/knip/test/plugins/sst.test.ts

```diff
@@ -0,0 +1,97 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwdEmptyConfig = resolve('fixtures/plugins/sst/empty-config');
+const cwdOneHandler = resolve('fixtures/plugins/sst/one-handler');
+const cwdOneHandlerInSrcLambdas = resolve('fixtures/plugins/sst/one-handler-in-lambdas');
+const cwdOneHandlerInSrcHandlers = resolve('fixtures/plugins/sst/one-handler-in-src');
```

Comment:
Maybe good to move each `cwd` to each test context individually to hold them together.